cmake/Platform/Mac/InstallUtils_mac.cmake.in

```diff
@@ -130,27 +130,50 @@ endfunction()
 
 function(ly_copy source_file target_directory)
     
-    if("${source_file}" MATCHES "\\.[Ff]ramework[^\\.]")
+    set(is_framework FALSE)
+    if("${source_file}" MATCHES "\\.[Ff]ramework")
 
         # fixup origin to copy the whole Framework folder
         string(REGEX REPLACE "(.*\\.[Ff]ramework).*" "\\1" source_file "${source_file}")
+        set(is_framework TRUE)
 
     endif()
-    get_filename_component(target_filename "${source_file}" NAME)
-    file(COPY "${source_file}" DESTINATION "${target_directory}" FILE_PERMISSIONS @LY_COPY_PERMISSIONS@ FOLLOW_SYMLINK_CHAIN)
-
-    # Our Qt and Python frameworks aren't in the correct bundle format to be codesigned.
-    if("${target_filename}" MATCHES "(Qt[^.]+)\\.[Ff]ramework")
-        fixup_qt_framework(${CMAKE_MATCH_1} "${target_directory}/${target_filename}")
-        # For some Qt frameworks(QtCore), signing the bundle doesn't work because of bundle
-        # format issues(despite the fixes above). But once we've patched the framework above, there's
-        # only one executable that we need to sign so we can do it directly.
-        set(target_filename "${target_filename}/Versions/5/${CMAKE_MATCH_1}")
-    elseif("${target_filename}" MATCHES "Python.framework")
-        fixup_python_framework("${target_directory}/${target_filename}")
-        codesign_python_framework_binaries("${target_directory}/${target_filename}")
+
+    cmake_path(GET source_file FILENAME target_filename)
+    cmake_path(APPEND full_target_directory "$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}" "${target_directory}")
+    cmake_path(APPEND target_file "${full_target_directory}" "${target_filename}")
+ 
+    if(NOT is_framework)
+        file(SIZE "${source_file}" source_file_size)
+        if(NOT is_framework AND EXISTS "${target_file}")
```

Comment:
Nit: We already checked NOT is_framework above. So we don't need to check here again.